pom.xml

```diff
@@ -36,15 +36,15 @@ POSSIBILITY OF SUCH DAMAGE.
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>gov.nasa.pds</groupId>
-  <artifactId>registry</artifactId>
+  <artifactId>registry-mgr-legacy</artifactId>
   <version>2.3.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>Registry</name>
+  <name>Legacy Registry Manager Solr</name>
   <description>
-    The Registry provides provides the PDS-specific search protocol and the search capability for the PDS search index generated through the Search Core software. The core functionality for this service is satisfied by Apache Solr.
+    The Legacy Registry provides provides the PDS-specific search protocol and the search capability for the PDS search index generated through the Search Core software. The core functionality for this service is satisfied by Apache Solr.
```

Comment:
From the changes in the README.md: replace "The core functionality for this service is satisfied by Apache Solr." with "Future implementations are already underway for the PDS Search API." (or append this "Future implementations..." sentence to the existing paragraph)

Reply:
@c-suh feel free to update. I do not think this holds up this PR.